app/jobs/sitemap_job.rb

```diff
@@ -11,18 +11,19 @@ def perform
       add '/page/faq', priority: 0.3
       add '/page/tutorial', priority: 0.3
       add '/disclaimer', priority: 0.3
-      add '/privacy', priority: 0.3
-      add '/copyright', priority: 0.3
       add '/right-to-information', priority: 0.3
```

Comment:
I think we can remove /disclaimer, /right-to-information and /website-accessibility as well. Neither of those pages exist on the Amplify site, and those footer URLs point to www.sl.nsw.gov.au.